<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysCommandDialog</class>
 <widget class="QDialog" name="SysCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>491</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>System commands</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>70</height>
      </size>
     </property>
     <property name="title">
      <string>Log directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="directoryButton">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="directoryEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Command sessions</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Command list</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QToolButton" name="addCommandButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../glogg.qrc">
              <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QListWidget" name="commandListWidget"/>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="removeCommandButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../glogg.qrc">
              <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QPushButton" name="runButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Session</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
          <item row="2" column="1">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Initial regex</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="regexButton">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="regexEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Command to execute</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QPushButton" name="executeButton">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="executeLineEdit"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="repeatOnFailCheck">
            <property name="text">
             <string>Repeat on failure</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QCheckBox" name="appendLogFileCheck">
            <property name="text">
             <string>Append log file while repeating</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QCheckBox" name="keepFileAfterClosingSessionCheck">
            <property name="text">
             <string>Keep log file after closing session</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="OkCloseButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox</zorder>
  <zorder>OkCloseButtonBox</zorder>
 </widget>
 <resources>
  <include location="../glogg.qrc"/>
 </resources>
 <connections/>
</ui>
